doc/source/visualizing/geographic_projections_and_transforms.rst

```diff
@@ -0,0 +1,86 @@
+.. _geographic_projections_and_transforms:
+
+Geographic Projections and Transforms
+=====================================
+
+Geographic data that is on a sphere can be visualized by projecting that data
+onto a representation of that sphere flattened into 2d space. There exist a
+number of projection types, which can be found in the `the cartopy
+documentation <https://scitools.org.uk/cartopy/docs/v0.15/crs/projections.html>`_.
+yt now supports these projection types for geographically loaded data.
+Underlying data is assumed to have an underlying projection type of `PlateCarree`. If
```

Comment:
Make this a link to the docs for `PlateCaree`, or at least note that `PlateCaree` means that it's lat/lon grid?